### PR TITLE
#417 Showing pdf invoice in a new tab.

### PR DESCRIPTION
**Pull request by criske**
PR for #417

screen

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by criske** [comment]
@amihaiemil no bounty needed, it was a short fix.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

File: src/main/java/com/selfxdsd/selfweb/api/ContributorApi.java [hunk 1]
```diff
@@ -220,8 +220,9 @@ public ContributorApi(final User user) {
         @PathVariable final String owner,
         @PathVariable final String name,
         @PathVariable final int invoiceId,
-        @RequestParam("role") final String role
-    ) throws IOException {
+        @RequestParam("role") final String role,
+        @RequestParam(value = "view", required = false) final boolean view
```
Comment (amihaiemil) [inline src/main/java/com/selfxdsd/selfweb/api/ContributorApi.java:224]: @criske Why is this parameter needed? :D 

There are a bit more changes than I expected.

**Review by amihaiemil — CHANGES_REQUESTED** [review]
@criske see comment :D

**Comment by amihaiemil** [comment]
@criske scuze, acuma vad gif-u.

Eu as vrea un singur buton, ca si pana acum, doar ca in loc sa iti deschida direct fereastra de download, ti-l deschide in tab separat, de unde poti da download.

**Comment by criske** [comment]
@amihaiemil A pai atunci pr devine si mai simplu, trebuie doar sa fac "content-disposition"-ul inline, iar la link target-uri sa trec "_blank"

**Comment by amihaiemil** [comment]
@criske mersi. Ideea initiala cu doua iconite nu e rea, imi place iconita cu cloud, dar e putin redundant. Ca el dupa ce e dus pe tab-ul cu PDF-ul, are acolo direct optiuni de download/print/whatever.

Si oricum, asta era o varianta doar pt tabul de Contracts de pe pagina de proiect. In tabelul contirbutorului (de pe Contributor Dashboard) ar fi aratat putin aiura, pt ca lui acolo ii apare si Platform Invoice-ul (iconita visinie), dupa ce e platita factura.

**Comment by amihaiemil** [comment]
@rultor merge

**Pull request merged**